robyn/robyn.pyi

```diff
@@ -49,6 +49,10 @@ class Url:
     host: str
     path: str
 
+@dataclass
+class Identity:
+    claims: dict[str, str]
```

Comment:
I would keep this as a dataclass, it feels more natural to me. Otherwise we would have a dict inside a dict which is something I find confusing :sweat_smile: